stubs/python-gnupg/gnupg/__init__.pyi

```diff
@@ -0,0 +1,257 @@
+import logging
+import re
+from typing import Any
+
+__version__: str = ...
+__author__: str = ...
+__date__: str = ...
+
+from typing import Protocol
+
+string_types: type[str] = ...
+text_type: type[str] = ...
+path_types: tuple[type[str]] = ...
+
+log_everything: bool = ...
+logger: logging.Logger = ...
+fsencoding: str = ...
+
+UNSAFE: re.Pattern[Any] = ...
```

Comment:
Considering this is only defined on some platforms, I would just add `gnupg.UNSAFE` to `@tests/stubtest_allowlist.txt`.